source/includes/ios/_ios.md

```diff
@@ -227,6 +258,9 @@ Set the frequency of the updates from the accelerometer.
 
 Below are all available commands relating to Bluetooth within the **SensumSDK**.
 
+* **SensumKit** supports connecting to BLE devices for reading heart rate measurements. For a list of tested compatible devices please view the <a href = "http://help.sensum.co/knowledge_base/topics/what-type-of-sensors-can-i-use"> list of compatible devices</a> at our Knowledge Centre.
```

Comment:
Might be worth clarifying the relationship between **SensumSDK** and **SensumKit**, i.e. that Kit is specific to iOS

Reply:
Added a Glossary of Terms to clarify these.